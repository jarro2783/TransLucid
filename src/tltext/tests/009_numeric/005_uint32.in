%%
uint32"0";;
uint32"1";;
uint32"2";;
uint32"4294967295";;

uint32"100" + uint32"200";;

uint32"60000" - uint32"200";;

uint32"5" * uint32"3";;

uint32"25" % uint32"7";;

uint32"24" / uint32"7";;
