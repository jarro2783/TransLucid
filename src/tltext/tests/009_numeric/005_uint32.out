uint32"0"
uint32"1"
uint32"2"
uint32"4294967295"
uint32"300"
uint32"59800"
uint32"15"
uint32"4"
uint32"3"
